Automattic/wp-calypso
# Fixes ordering by views in the Advertising dashboard redesing

Related to #SELFSERVE-544

## Proposed Changes

We are using the wrong value for the order_by key of views. This PR changed the value to the correct one `monthly_view_count`.

I also fixed a typescript error we had in the `getPagedBlazeSearchData` method, it was using an incorrect Campaign type reference (i1 instead of i2).

## Testing Instructions

* Check out this branch or load the live preview activating the `promote-post/redesign-i2` flag
* Open the wp-admin page
* Go to Tools->Advertising
* Change the order filter to **Most viewed**. 
* Check that the list gets ordered by post's view count.

## Pre-merge Checklist
